features/polyExplorer/src/polyExplorer.jsx

```diff
@@ -67,16 +58,11 @@ const PolyExplorerApp = () => {
                     <ExampleStory />
                 </Route>
             </Switch>
-            {firstRun && (
-                <OnboardingPopup
-                    onClose={handleOnboardingPopupClose}
-                    onMoreInfo={handleOnboardingPopupMoreInfo}
-                />
-            )}
             {popUp &&
                 popUp.component({
-                    onClose: closePopUp,
+                    onClose: popUp.onClose || closePopUp,
```

Comment:
I don't think it's possible now that popUp.onClose does not exist, right? Every possible creation of popUp adds onClose... Or maybe not. `info-popup` maybe does not. Anyway, Shouldn't this be done when the popUp is created, not here?

Reply:
Yeah better there, you're right